netbox_custom_objects/models.py

```diff
@@ -164,9 +164,15 @@ class CustomObjectType(PrimaryModel):
     _through_model_cache = (
         {}
     )  # Now stores {custom_object_type_id: {through_model_name: through_model}}
-    name = models.CharField(max_length=100, unique=True)
+    name = models.CharField(
+        max_length=100,
+        unique=True,
+        help_text=_("Internal lowercased object name, e.g. \"vendor_policy\""),
+    )
```

Comment:
It says lowercased but there is no validation or conversion on this, should we enforce that?